sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/WebPubSubHub.cs

```diff
@@ -30,6 +30,21 @@ public virtual ValueTask<ConnectEventResponse> OnConnectAsync(ConnectEventReques
             return default;
         }
 
+        /// <summary>
+        /// MQTT connect event method.
+        /// </summary>
+        /// <param name="request"><see cref="MqttConnectEventRequest"/> to get client connect request information.</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> used to cancel the request.</param>
+        /// <returns>
+        /// <para>If you want to accept the connection, return an <see cref="MqttConnectEventRequest.CreateMqttResponse(string, System.Collections.Generic.IEnumerable{string}, System.Collections.Generic.IEnumerable{string})"/>.</para>
+        /// <para>If you want to reject the connection, return an <see cref="MqttConnectEventRequest.CreateMqttV311ErrorResponse(MqttV311ConnectReturnCode, string?)"/> or <see cref="MqttConnectEventRequest.CreateMqttV50ErrorResponse(MqttV500ConnectReasonCode, string?)"/></para>
+        /// </returns>
+        /// <remarks>If you don't override this method, MQTT "connect" events are handled by <see cref="OnConnectAsync(ConnectEventRequest, CancellationToken)"/> method.</remarks>
```

Comment:
typo here: `OnMqttConnectAsync(MqttConnectEventRequest request, CancellationToken cancellationToken)`, or maybe simply explain as `are handled by default return`?

Reply:
It's a feature by design... If users want to handle Web PubSub client and MQTT client `connect` events with unified logic, they can just implement the `OnConnectAsync` method .

Reply:
get it.